sklearn/feature_selection/tests/test_mutual_info.py

```diff
@@ -43,15 +47,15 @@ def test_compute_mi_cc():
     I_theory = np.log(sigma_1) + np.log(sigma_2) - 0.5 * np.log(np.linalg.det(cov))
 
     rng = check_random_state(0)
-    Z = rng.multivariate_normal(mean, cov, size=1000)
+    Z = rng.multivariate_normal(mean, cov, size=1000).astype(global_dtype)
 
     x, y = Z[:, 0], Z[:, 1]
 
     # Theory and computed values won't be very close, assert that the
     # first figures after decimal point match.
     for n_neighbors in [3, 5, 7]:
         I_computed = _compute_mi(x, y, False, False, n_neighbors)
-        assert_almost_equal(I_computed, I_theory, 1)
+        assert_allclose(I_computed, I_theory, 1)
```

Comment:
According to the comment, we should use atol here. rtol=1 means we allow 100% relative error.